game-app/game-core/src/main/java/games/strategy/engine/player/Player.java

```diff
@@ -39,8 +38,9 @@ public interface Player extends IRemote {
   @RemoteActionCode(6)
   String getName();
 
-  @RemoteActionCode(8)
-  PlayerTypes.Type getPlayerType();
```

Comment:
I'm 99% sure this is safe because this type seems to have been changed in the past without problems, and it is not serializable. So serialization over the network would have failed anyways.

Reply:
A 2.5 client connecting to a 2.6 server might exercise this. If that doesn't, then this method is seemingly never invoked.

Reply:
I think 2.5 hits an npe connecting to a 2.6 server right now

Reply:
@asvitkine Would you mind sharing a stacktrace of the error?

Reply:
@asvitkine Hmm I can run the code without problems in my test scenario which should cover the same code. However I noticed something that could be mistaken for a NPE.
The `UnifiedInvocationHandler` sometimes logs a stacktrace (for debug reference not because there is an actual exception) when the UI thread calls a blocking network operation:
```java
      log.info("Blocking network operation performed from EDT", new Exception());
```

I noticed that gets logged in lobby chat for example. The cleanest fix would be to run it in a non-ui thread, the dirty fix would be to configure the logger to make an exception for this class

Reply:
It's this issue: https://github.com/triplea-game/triplea/issues/10028

Note: It's a bug in 2.5, since the NPE happens for the 2.5 client.